node/test_helpers.go

```diff
@@ -11,6 +16,57 @@ import (
 
 var genesisValidatorKey = ed25519.GenPrivKey()
 
+type MockTester struct {
+	t *testing.T
+}
+
+func (m MockTester) Fail() {
+	fmt.Println("Failed")
+}
+
+func (m MockTester) FailNow() {
+	fmt.Println("MockTester FailNow called")
+}
+
+func (m MockTester) Logf(format string, args ...interface{}) {
+	fmt.Println("MockTester Logf called")
+}
+
+func (m MockTester) Errorf(format string, args ...interface{}) {
+	//fmt.Printf(format, args...)
+	fmt.Println("Errorf called")
+}
+
+func waitForFirstBlock(node *FullNode) error {
+	return testutils.Retry(300, 100*time.Millisecond, func() error {
+		if node.Store.Height() >= 1 {
+			return nil
+		}
+		return fmt.Errorf("node at height %v, expected >= 1", node.Store.Height())
+	})
```

Comment:
you can call `waitForAtLeastNBlocks`

```suggestion
	return waitForAtLeastNBlocks(node, 1)
```